src/compose/__main__.py

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: 2023 CERN (home.cern)
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""CLI to generate content for ohwr.org."""
+
+import argparse
+import logging
+
+import yaml
+from config import ConfigError, ProjectConfig
+from sources import ProjectSources
+
+parser = argparse.ArgumentParser()
+parser.add_argument('config', type=argparse.FileType('r'))
+args = parser.parse_args()
+
+config = yaml.safe_load(args.config)
+
+logging.basicConfig(
+    level=getattr(logging, config['log_level']),
+    format='%(asctime)s - %(levelname)s - %(message)s',  # noqa: WPS323
+)
+
+for project in config['projects']:
+    try:
+        project_config = ProjectConfig.from_url(**project)
+    except ConfigError as error:
+        msg = 'Could not configure the {0} project:\n↳ {1}'
```

Comment:
Did you forget the format arguments here? '0' and '1' should refer to something.

Also, I personally prefer python f'strings for this sort of things, but that's a matter of taste.

Reply:
I stop reporting this type of error here. Please fix it everywhere (it's in many places in the rest of the code as well) or explain to me why this works.

Reply:
'0' and '1' are positional arguments for the str.format() method.
The format method is called in the line bellow - `logging.error(msg.format(proj['id'], error))`.